Source.md

```diff
@@ -26,3 +26,8 @@ title: Flutter Interview Questions
 * What is the role of a Change Notifier Provider?
 * What is a ValueListenableBuilder?
 * What is an AspectRatio?
+* How would you access Stateful Widget properties from it's State?
+* What is InheritedWidget in Flutter?
+* Is there a suggested limit to the number of FloatingActionButtons a screen can have?. Give reason(s) for your answer
+* What is a Future in Dart?
```

Comment:
I think this specific question is repeated in line 41:

> Explain async, await and Future?

Reply:
Good catch @lucalves. Will get rid of that

Reply:
I think it's perfect now, @johngorithm.